docs/guides/tutorial.md

```diff
@@ -52,9 +52,15 @@ $ node
 In the file `createcredential.js`, we have a simple node `express` server. In the setup phase, we will use the private key, and the DID we created above; there is already an example key pair available in the file, but this should be replaced with your own keypair for any real application. Following along in the code, the first step is to create a `Credentials` object which will encapsulate our newly created keypair.
 
 ```js
+import { Resolver } from 'did-resolver'
+import { getResolver } from 'ethr-did-resolver'
+const providerConfig = { rpcUrl: 'https://mainnet.infura.io/<YOUR INFURA PROJECT ID>' }
+const resolver = new Resolver(getResolver(providerConfig))
+
 var credentials = new uport.Credentials({
   did: 'did:ethr:0xbc3ae59bc76f894822622cdef7a2018dbe353840',
-  privateKey: '74894f8853f90e6e3d6dfdd343eb0eb70cca06e552ed8af80adadcc573b35da3'
+  privateKey: '74894f8853f90e6e3d6dfdd343eb0eb70cca06e552ed8af80adadcc573b35da3',
```

Comment:
`signer: SimpleSigner(...)` here also. Are you sure you want an actual hard coded private key here in an example?

Reply:
This private key has been compromised the moment it touched git.
A few lines above there are instructions on how to store the key.